ballerina/tests/listener_client_tests.bal

```diff
@@ -986,3 +987,33 @@ function listenerWithPollTimeoutConfigTest() returns error? {
     check configListener.gracefulStop();
     test:assertEquals(receivedTimeoutConfigValue, TEST_MESSAGE);
 }
+
+@test:Config {enable: true}
```

Comment:
```suggestion
@test:Config {}
```

Reply:
I added this in all the test cases to easily disable them using a replace and run only a single test